google-cloud-spanner/lib/google/cloud/spanner/client.rb

```diff
@@ -582,13 +605,13 @@ def snapshot strong: nil, timestamp: nil, staleness: nil
         #
         #   db = spanner.client "my-instance", "my-database"
         #
-        #   users_types = db.types_for "users"
+        #   users_types = db.fields_for "users"
         #   db.insert "users", [{ id: 1, name: "Charlie", active: false },
         #                       { id: 2, name: "Harvey",  active: true }],
         #             types: users_types
         #
-        def types_for table, pairs: false
-          execute("SELECT * FROM #{table} WHERE 1 = 0").types pairs: pairs
+        def fields_for table
```

Comment:
where is this used? You can also query information schema to get this information.